Source/WebCore/platform/graphics/skia/NativeImageSkia.cpp

```diff
@@ -61,7 +61,12 @@ bool PlatformImageNativeImageBackend::hasAlpha() const
 
 DestinationColorSpace PlatformImageNativeImageBackend::colorSpace() const
 {
-    notImplemented();
+    auto colorSpace = platformImage()->refColorSpace();
+    if (!colorSpace)
+        colorSpace = platformImage()->imageInfo().refColorSpace();
```

Comment:
This is the same, `SkImage::refColorSpace()` returns `fInfo.refColorSpace()`, and `SkImage::imageInfo()` returns `fInfo`.